.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://postgres:Xboxnam101@localhost:5432/nerpm?schema=public"
```

Comment:
This should not be in the codebase. You have exposed your local database password to the public.